packages/blockchain-link/src/workers/solana/fee.ts

```diff
@@ -0,0 +1,37 @@
+import { Connection, Message, PublicKey } from '@solana/web3.js';
+import BigNumber from 'bignumber.js';
+
+const DEFAULT_COMPUTE_UNIT_PRICE = 100_000; // micro-lamports, value taken from other wallets
+const DEFAULT_COMPUTE_UNIT_LIMIT = 50_000; // sending tokens with token account creation requires ~28K units
```

Comment:
IIRC, this is fine, but while doing some reading I came across [this article](https://solana.com/developers/guides/advanced/how-to-use-priority-fees) where solana recommends "To best optimize our transaction, request exactly <number> CU with the Compute Budget Program when adding additional priority fees."
I'm not sure what the reason is behind that as I think only as many CU as needed are used anyway. 
Could you quickly test this?

Reply:
> I think only as many CU as needed are used anyway

If you reserve 500K CUs, you pay for 500K CUs even if your transaction only uses 500.

Thanks for the comment though, because it pushed to revisit this and in the end I also implemented transaction simulation into the priority fee computations and so the limit will be exactly what the transaction needs + 20%. [This article](https://solana.com/developers/guides/advanced/how-to-request-optimal-compute) mentions a buffer of 10%, so we add 20% just to be sure.

However I'm not 100% convinced about this approach since e.g. SOL transfer now adds 18 lamports as fee i.e. the final fee is 5018 lamports instead of 5000 and I'm not entirely sure that's enough :D I'll discuss with Trezor guys, the previous approach is still implemented if we drop the last two commits.